api/src/main/java/com/velocitypowered/api/bossbar/BossBar.java

```diff
@@ -0,0 +1,145 @@
+package com.velocitypowered.api.bossbar;
+
+import com.velocitypowered.api.proxy.Player;
+import java.util.Collection;
+import java.util.UUID;
+import net.kyori.text.Component;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+/**
+ * Represents a boss bar, which can be send to a (group of) player(s) <b>WARNING!!!</b> This works
+ * only on 1.9+. <b>YOU ARE ON YOUR OWN IF YOU USE THIS ON 1.8</b>
+ */
+public interface BossBar {
+
+  /**
+   * Adds all specified players to this boss bar
+   *
+   * @param players players
+   * @see #addPlayer(Player)
+   */
+  void addPlayers(@NonNull Iterable<Player> players);
```

Comment:
`@NonNull` is the default in Checker Framework, so specifying that explicitly is totally redundant.